### PR TITLE
Minor updates to create rc script

### PR DESCRIPTION
**Pull request by nick-stroud**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: tools/create-release-candidate.sh [hunk 1]
```diff
@@ -113,4 +114,4 @@ echo
 echo
 echo "Consider running the test babysitter using the pull request number from above:"
 echo
-echo "babysit.py --pr ${PR_NUM} --all -c 1"
+echo "tools/cloud-build/babysit/run --pr <PR_NUM> --all -c 1 --project <team-project>"
```
Comment (mr0re1) [inline tools/create-release-candidate.sh:117]: Thank you for fixing the pass, but why did you make it non-copy-paste-ready ?

Comment (mr0re1) [reply tools/create-release-candidate.sh:117]:
```sh
echo "tools/cloud-build/babysit/run --pr ${PR_NUM} --all -c 1"
```

Comment (mr0re1) [inline tools/create-release-candidate.sh:117]: Please remove ` --project <team-project>` , AFAIK it doesn't add to usability.

**Review by mr0re1 — APPROVED** [review]

File: tools/create-release-candidate.sh (hunk 1 above)
Comment (nick-stroud) [reply tools/create-release-candidate.sh]: Done

**Pull request merged**